src/js/index.js

```diff
@@ -0,0 +1,130 @@
+function numberWithCommas(num) {
+  return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+}
+
+function showFirst(...els) {
+  var el, found, i, len, results
+  found = false
+  results = []
+  for (i = 0, len = els.length; i < len; i++) {
+    el = els[i]
+    if (el && !found) {
+      found = true
+      results.push((el.style.display = 'block'))
+    } else if (el) {
+      results.push((el.style.display = 'none'))
+    } else {
+      results.push(void 0)
+    }
+  }
+  return results
+}
+
+function hide(...els) {
+  var el, i, len
+  const results = []
+  for (i = 0, len = els.length; i < len; i++) {
+    el = els[i]
+    if (el) {
+      results.push((el.style.display = 'none'))
+    }
+  }
+  return results
+}
+
+function setMinVersion(version = 11) {
+  version = parseInt(version)
+  const ref = document.querySelectorAll('.comparison')
+  const results = []
+  for (let i = 0, len = ref.length; i < len; i++) {
+    const section = ref[i]
+    const blocks = section.querySelectorAll('.browser')
+    const versions = {}
+    for (let j = 0, len1 = blocks.length; j < len1; j++) {
+      const block = blocks[j]
+      versions[block.getAttribute('data-browser')] = block
+    }
+    switch (version) {
+      case 8:
+        showFirst(versions['ie8'])
+        results.push(hide(versions['ie9'], versions['ie10']))
+        break
+      case 9:
+        showFirst(versions['ie9'], versions['ie8'])
+        results.push(hide(versions['ie10']))
+        break
+      case 10:
+        showFirst(versions['ie10'], versions['ie9'], versions['ie8'])
+        results.push(hide(versions['ie11']))
+        break
+      case 11:
+        results.push(
+          showFirst(
+            versions['ie11'],
+            versions['ie10'],
+            versions['ie9'],
+            versions['ie8']
+          )
+        )
+        break
+      default:
+        results.push(void 0)
+    }
+  }
+  return results
+}
+
+function filter(term) {
+  let visibleIndex = 0
+  let allEmpty = true
+  const ref = document.querySelectorAll('section')
+  for (const i = 0, len = ref.length; i < len; i++) {
+    const section = ref[i]
+    let empty = true
+    const ref1 = section.querySelectorAll('.comparison')
+    for (const j = 0, len1 = ref1.length; j < len1; j++) {
```

Comment:
This is a constant variable, so it won't allow reassignment. Should be `let` instead
```suggestion
    for (let j = 0, len1 = ref1.length; j < len1; j++) {
```